### DISCUSSION TITLE
Several Questions

### DISCUSSION THREAD

**Discussion by MeowCata** [risinek/esp32-wifi-penetration-tool#103] [Q&A] [answered]
First,I'd like to thank the developers
But, I'd like to ask,how can I change the ssid orpassword of ManagementAP

**Comment by risinek** [comment, marked as answer]
change it here and rebuild https://github.com/risinek/esp32-wifi-penetration-tool/blob/e346e28ed9c93e98ab686c30eb8c4bc549313ba3/components/wifi_controller/Kconfig#L9

**Reply by MeowCata** [reply]
thanks a lot

**Reply by Sanupawanni** [reply]
how to rebuild?
